### PR TITLE
new highlight message type for CLI init success message & console link fix

### PR DESCRIPTION
**Pull request by okane16**

### CODE REVIEW

**Review by tg339 — APPROVED** [review]
LGTM ship it!!!

File: apps/framework-cli/src/cli.rs [hunk 1]
```diff
@@ -152,33 +152,48 @@ async fn top_command_handler(
...
                     }
 
-                    show_message!(
-                        MessageType::Success,
-                        Message {
-                            action: "Created".to_string(),
-                            details: "new project".to_string(),
-                        }
-                    );
+                    {
```
Comment (callicles) [inline apps/framework-cli/src/cli.rs:163]: Why did you put it in a closure?

It doesn't feel like that should be necessary

**Review by callicles — COMMENTED** [review]
Nice first `rust` PR! You could merge as is, curious about, what looks like extra closures around the `show_message!`

File: apps/framework-cli/src/cli.rs (hunk 1 above)
Comment (okane16) [reply apps/framework-cli/src/cli.rs:163]: @tg339 can better explain but it was basically the fastest workaround for avoiding duplicate TERM imports with the back to back !show_message macros

**Pull request merged**